src/main/resources/collaboratorPage.json

```diff
@@ -1,4 +1,12 @@
 {
+  "metadata": {
+    "pagination": {
```

Comment:
Could you add the description on what do you want to do on this issue? Is this change the only thing necessary? Do you have tests or screenshot to reflect this change? I am not sure I understand 🙈

Reply:
I have added a description with a screenshot of the /collaborators page when I run locally after the change